pss/types.go

```diff
@@ -117,6 +118,18 @@ func (m *msgParams) Bytes() (paramBytes []byte) {
 	return paramBytes
 }
 
+type outboxMsg struct {
+	msg       *PssMsg
+	startedAt time.Time
```

Comment:
What's the plan with this `outboxMsg`?

Reply:
We keep it as is for now, to measure how long it takes to process a message after it has been `sent`. I think this is useful at least initially when we roll out `push sync`. It has no other purpose but to bundle a `startedAt` timestamp with the message.

Reply:
But the extra information is not being used currently, right?

Reply:
It is used in the timers that we emit, so that we know if we are making progress with these changes or not.

See:
```
metrics.GetOrRegisterResettingTimer("pss.handle.outbox", nil).UpdateSince(msg.startedAt)
```